lib/plugins/helper/open_graph.js

```diff
@@ -116,10 +116,6 @@ function openGraphHelper(options = {}) {
   }
 
   result += meta('twitter:card', twitterCard);
-  result += meta('twitter:title', title);
-  if (description) {
-    result += meta('twitter:description', description, false);
-  }
 
   if (images.length) {
     result += meta('twitter:image', images[0], false);
```

Comment:
`twitter:image` can deletable as well. Isn't it?

Reply:
@YoshinoriN It seems like `og:image` support multiple images while `twitter:image` only support one image. The docs from twitter do not mention multiple images usage.

Reply:
[The doc](https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/markup.html) is not exactly clear.

> URL of image to use in the card. Images must be less than 5MB in size. JPG, PNG, WEBP and GIF formats are supported. Only the first frame of an animated GIF will be used. SVG is not supported.

I lean on single twitter:image and keeping it.

Reply:
@SukkaW @curbengh 


I found following comments from twitter developers forum.

* https://twittercommunity.com/t/for-multiple-images-twitter-is-not-following-the-ogp-spec/15875 
* https://twittercommunity.com/t/twitter-cards-i-have-a-problem-help-me/15320/98
* https://twittercommunity.com/t/wrong-image-appearing-in-twitter-cards-using-wordpress-seo-by-yoast-plugin/15747/10

According to them. The twitter card use last `og:image` if some `og:image` tag exists. And it not following [ogp specification](https://ogp.me/#array).

They are very old comments (4~6 years ago). I could not find more latest. But, IMHO we should use `twitter:image` if we can not find more information about this matter.

I will merge this if other opinion does not exist after wait a while.

Reply:
> According to comments. The twitter card use last og:image if multiple og:image tags exists. It's not following ogp specification.

I figure most users would expect the first image, which is the current behavior of `open_graph.js`. Let's keep `twitter:image` for now.